src/dftbp/api/mm/dftbplus.h

```diff
@@ -278,6 +278,26 @@ void dftbp_get_energy(DftbPlus *instance, double *mermin_energy);
 void dftbp_get_gradients(DftbPlus *instance, double *gradients);
 
 
+/**
+ * Queries the number of basis functions for each atom in current geometry.
+ *
+ * \param[inout] instance Handler of the DFTB+ instance.
+ *
+ * \param[out] nOrbitals number of orbitals on each atom
+ */
+void dftbp_get_nr_basisfns(DftbPlus *instance, int *nOrbitals);
```

Comment:
```suggestion
void dftbp_get_nr_orbitals(DftbPlus *instance, int *norbitals);
```
(Or is there any reason to call it differently as we do in the Fortran part (basisfns vs. orbitals)?